middlewares/error_pages.go

```diff
@@ -0,0 +1,75 @@
+package middlewares
+
+import (
+	"net/http"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/containous/traefik/log"
+	"github.com/containous/traefik/types"
+	"github.com/vulcand/oxy/forward"
+	"github.com/vulcand/oxy/utils"
+)
+
+//ErrorPagesHandler is a middleware that provides the custom error pages
+type ErrorPagesHandler struct {
+	HTTPCodeRanges     [][2]int
+	BackendURL         string
+	errorPageForwarder *forward.Forwarder
+}
+
+//NewErrorPagesHandler initializes the utils.ErrorHandler for the custom error pages
+func NewErrorPagesHandler(errorPage types.ErrorPage, backendURL string) (*ErrorPagesHandler, error) {
+	fwd, err := forward.New()
+	if err != nil {
+		return nil, err
+	}
+
+	//Break out the http status code ranges into a low int and high int
+	//for ease of use at runtime
+	var blocks [][2]int
+	for _, block := range errorPage.Status {
+		codes := strings.Split(block, "-")
+		lowCode, err := strconv.Atoi(codes[0])
+		if err != nil {
+			return nil, err
+		}
+		highCode, err := strconv.Atoi(codes[1])
```

Comment:
I think there are actually two options here, both simple.  1) catch it and log an error or 2) catch it and assume the user means to serve the error page for just the single code.  In this case the single entry can still be used to create the correct array for checking at runtime.  So, for example if ["503"] (instead of ["503-503"]) is configured,  the array ["503", "503"] is created and we continue on.  

This was actually raised by @nmengin above, I should have just addressed it then 

WDYT @Juliens ?